.doctrine-project.json

```
@@ -8,11 +8,7 @@
         {
             "name": "2.0",
             "branchName": "2.0.x",
-            "slug": "current",
-            "aliases": [
-                "latest",
-                "stable"
-            ],
+            "aliases": ["latest"],
```

Let's keep the slug :snail: 

```suggestion
            "slug": "2.0",
            "aliases": [ "latest"],
```